inc/composer/class-docker-compose-generator.php

```diff
@@ -653,6 +653,15 @@ protected function get_config( $module = 'local-server' ) : array {
 		return array_merge( $defaults, $config );
 	}
 
+	/**
+	 * Check if the current system is ARM64.
+	 *
+	 * @return bool
+	 */
+	protected function is_arm64() : bool {
+		return strpos( php_uname(), 'aarch64' ) !== false;
```

Comment:
I think you can pass `'m'` to `php_uname()` to access this value directly eg:
```suggestion
		return php_uname( 'm' ) === 'aarch64';
```